pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -10,25 +10,29 @@
 }:
 
 let
-  archUrl = name: arch: "https://downloads.vivaldi.com/stable/${name}_${arch}.deb";
-in
-stdenv.mkDerivation rec {
-  version    = "1.1";
-  debversion = "stable_1.1.453.47-1";
-  product    = "vivaldi";
-  name       = "${product}-${version}";
+  isLinux64 = stdenv.system == "x86_64-linux";
```

Comment:
I don't really care for the branching. You are right in that `meta.platform` already defines that this is linux-only.
So using `stdenv.is64bit` should do the job.